prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -88,20 +86,13 @@ data class Hl7Configuration
             else -> ""
         }
         return mapOf(
-            "processing_mode_code" to (if (useTestProcessingMode) "T" else "P"),
+            "processing_mode_code" to (processingModeCode ?: "P"),
```

Comment:
Why not just use a default value in the schema for this?    I'm doing that on the sender side, for this very field.  However, this does require separate schemas for each sender -- that probably scales ok-ish on the sender side, but not very well on the receiver side.